Xiaomi/MiA3/res/values/notch.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+** Copyright 2017, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <!-- Height of the status bar in portrait -->
+    <dimen name="status_bar_height_portrait">35.0dip</dimen>
```

Comment:
It should be an absolute px value

Reply:
Overlay has been tested on real device since about a week. Auto brightness works, status bar seems a bit to large and there is only space for 3 notifications even though foru notifications could very well fit before the notch covers them.

Reply:
How am I able to get the absolute px value?

Reply:
> How am I able to get the absolute px value?

Preferably manually with some graphical mockups. Looks like the value should be around 53px.